src/pageObject.js

```diff
@@ -1,16 +1,39 @@
-'use strict';
+/* eslint-disable */
+"use strict";
 
 class Header {
-
-};
+  get logo() {
+    return "Conduit logo";
+  }
+}
 
 class PageObject {
-
-};
+  constructor(url, header = new Header()) {
+    this.header = header;
+    this.url = url;
+  }
+  clickOnLogo() {
+    return "Click on the " + this.header.logo;
+  }
+}
 
 class ArticlePage extends PageObject {
+  constructor(url, slug) {
+    super();
+    this.url = url + slug;
```

Comment:
Setting `this.url` here will overwrite the `url` set by the `PageObject` constructor. Consider appending the `slug` to `this.url` after calling `super(url)`.